portal-web/test/functional/com/liferay/portalweb/tests/enduser/documentmanagement/adaptivemedia/AdaptiveMedia.testcase

```diff
@@ -514,12 +514,42 @@ definition {
 	}
 
 	@description = "This test covers LPS-136280. It validates that an image can be adapted in a web content after it is updated."
-	@ignore = "Test Stub"
 	@priority = "4"
 	test ImageVariantsAreGeneratedAfterEditingWebContentWithImage {
+		property portal.acceptance = "false";
+
+		WebContentNavigator.openWebContentAdmin(siteURLKey = "guest");
+
+		WebContentNavigator.gotoAddCP();
```

Comment:
I think this can be replaced by WebContentNavigator.opentoBasicArticle();

Can you confirm @Jeremy-Ch ? I'm going to make the change but not test it

Reply:
@austinchiang I have tested it locally. We can replace it. thanks.
```
WebContentNavigator.openToAddBasicArticle(
			groupName = "Guest",
			siteURLKey = "guest");
```